ui/src/App.vue

```diff
@@ -44,6 +45,12 @@ export default Vue.extend({
       this.$store.dispatch('app/dismissMessage', message)
     },
   },
+  errorCaptured (err) {
+    if (err instanceof APIErrorAuthorization) {
+      this.$router.push({ name: 'NotAuthorized', params: { link: err.details.link.href } })
+    }
+    return false
```

Comment:
Won't that stop any error from propagating?

Reply:
True, I missread the documentation.

Reply:
Yes, I misread the documentation, I will fix that.